test/pkg/environment/common/expectations.go

```diff
@@ -34,11 +33,8 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-const TestLabelName = "testing.karpenter.sh/test-id"
-
 func (env *Environment) ExpectCreated(objects ...client.Object) {
 	for _, object := range objects {
-		object.SetLabels(lo.Assign(object.GetLabels(), map[string]string{TestLabelName: env.ClusterName}))
```

Comment:
Why are we removing these labels?

Reply:
They're added here now: https://github.com/aws/karpenter-core/pull/27. We could add in cluster name, but right now, our tests assume non-parallel (and probably need to to avoid provisioner collision), so the label value doesn't really matter.